libs/EXTERNAL/libtatum/tatum_test/main.cpp

```diff
@@ -94,6 +104,34 @@ Args parse_args(int argc, char** argv);
 double median(std::vector<double> values);
 double arithmean(std::vector<double> values);
 
+template<class I>
+double median(I begin, I end) {
+    std::sort(begin, end);
```

Comment:
There are faster (linear time) [algorithms](https://en.wikipedia.org/wiki/Selection_algorithm) for this.

Reply:
Agreed, but as this is non-critical test verification code not used by VTR I don't think its a major issue.